src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -180,19 +184,48 @@ private void ExpandCategory(TreeViewItem sender, NodeCategoryViewModel selectedC
             foreach (var categoryToBeCollapsed in allExpandedCategories)
             {
                 // If class button was clicked.
-                if (categoryToBeExpanded == null)
+                if (selectedClass.SubCategories.Count == 0)
                 {
                     var categoryClasses = categoryToBeCollapsed.Items[0] as ClassesNodeCategoryViewModel;
```

Comment:
It's code from NodeCategoryViewModel. When we insert some item.

```c#
// Classes must be first in any case.
if (entry is ClassesNodeCategoryViewModel)
       Items.Insert(0, entry);
```
So, I'm sure, Classes at the first place.